programs/compressed-token/program/docs/compressed_token/MINT_ACTION.md

```
@@ -214,19 +205,23 @@ Packed accounts (remaining accounts):
+- `ErrorCode::MintActionMissingMintAccount` (error code: 6048) - Missing SPL mint account when required
+- `ErrorCode::MintActionMissingTokenPoolAccount` (error code: 6049) - Missing token pool PDA when required
+- `ErrorCode::MintActionMissingTokenProgram` (error code: 6050) - Missing token program when required
+- `ErrorCode::MintActionInvalidExtensionIndex` (error code: 6059) - Extension index out of bounds
+- `ErrorCode::MintActionInvalidExtensionType` (error code: 6062) - Extension is not TokenMetadata type
+- `ErrorCode::MintActionMetadataKeyNotFound` (error code: 6063) - Metadata key not found for removal
+- `ErrorCode::MintActionMissingExecutingAccounts` (error code: 6064) - Missing required execution accounts
 - `ErrorCode::CpiContextExpected` (error code: 6085) - CPI context required but not provided
 - `AccountError::InvalidSigner` (error code: 12015) - Required signer account is not signing
 - `AccountError::NotEnoughAccountKeys` (error code: 12020) - Missing required accounts
+
+### Spl mint migration
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)